backend/fms_core/models/sample_metadata.py

```diff
@@ -0,0 +1,24 @@
+import reversion
+
+from .tracked_model import TrackedModel
+from django.db import models
+from .sample import Biosample
+
+from ._constants import STANDARD_NAME_FIELD_LENGTH
+from ._validators import name_validator
+
+
+@reversion.register()
+class SampleMetadata(TrackedModel):
+    """ Class to store additional sample metadata. """
+
+    name = models.CharField(max_length=2000, validators=[name_validator],
```

Comment:
not 2000 here ... only for value... we do not want 2000 character long field name ;).